### PR TITLE
Add FlowCutterPACE17

### PR DESCRIPTION
**Pull request by brenjohn**
Adds Flow Cutter PACE17: https://github.com/kit-algo/flow-cutter-pace17

### CODE REVIEW

File: F/FlowCutterPACE17/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd flow-cutter-pace17/
+$CXX -c -Wall -std=c++11 -O3 -DNDEBUG src/*.cpp
+$CXX -o flow_cutter_pace17 *.o
+install_license LICENSE
+mkdir $bindir 
+mv flow_cutter_pace17 $bindir/flow_cutter_pace17
```
Comment (giordano) [inline F/FlowCutterPACE17/build_tarballs.jl:21]: A bit simpler:
```suggestion
mkdir -p "${bindir}"
$CXX -o "${bindir}/flow_cutter_pace17${exeext}" *.o
install_license LICENSE
```
:wink:

File: F/FlowCutterPACE17/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd flow-cutter-pace17/
+$CXX -c -Wall -std=c++11 -O3 -DNDEBUG src/*.cpp
+mkdir -p "${bindir}"
+$CXX -o "${bindir}/flow_cutter_pace17${exeext}" *.o
+install_license LICENSE
```
Comment (giordano) [inline F/FlowCutterPACE17/build_tarballs.jl:20]: Can simplify even further (in this case the license should be installed automatically):
```suggestion
cd $WORKSPACE/srcdir/flow-cutter-pace17/
mkdir -p "${bindir}"
c++ -o "${bindir}/flow_cutter_pace17${exeext}" -Wall -std=c++11 -O3 -DNDEBUG src/*.cpp
```

**Comment by brenjohn** [comment]
Ah that's great! Thanks for the fix @giordano

**Comment by giordano** [comment]
Thanks a lot for your contribution!

**Pull request merged**